src/components/constants/Pokemon.ts

```diff
@@ -17,6 +17,11 @@ export interface IPokemonListing {
     height: [string, string];
 
     /**
+     * Storage for what item is being held.
+     */   
+    item: string;
```

Comment:
The Pokemon listings here shouldn't specify an item. This is the wrong interface to add it to.